[2023-12-19T04:38:56.970+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: steal_etl_storeToPostgres.insert_sql manual__2023-12-19T04:38:50.598977+00:00 [queued]>
[2023-12-19T04:38:56.979+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: steal_etl_storeToPostgres.insert_sql manual__2023-12-19T04:38:50.598977+00:00 [queued]>
[2023-12-19T04:38:56.980+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-19T04:38:56.992+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_sql> on 2023-12-19 04:38:50.598977+00:00
[2023-12-19T04:38:57.004+0000] {standard_task_runner.py:57} INFO - Started process 1602 to run task
[2023-12-19T04:38:57.007+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'steal_etl_storeToPostgres', 'insert_sql', 'manual__2023-12-19T04:38:50.598977+00:00', '--job-id', '386', '--raw', '--subdir', 'DAGS_FOLDER/steal_etl_storeToPostgres.py', '--cfg-path', '/tmp/tmpyn1g9yd4']
[2023-12-19T04:38:57.010+0000] {standard_task_runner.py:85} INFO - Job 386: Subtask insert_sql
[2023-12-19T04:38:57.067+0000] {task_command.py:416} INFO - Running <TaskInstance: steal_etl_storeToPostgres.insert_sql manual__2023-12-19T04:38:50.598977+00:00 [running]> on host f97daee24bd9
[2023-12-19T04:38:57.258+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='christinelee1114@gmail.com' AIRFLOW_CTX_DAG_OWNER='xin' AIRFLOW_CTX_DAG_ID='steal_etl_storeToPostgres' AIRFLOW_CTX_TASK_ID='insert_sql' AIRFLOW_CTX_EXECUTION_DATE='2023-12-19T04:38:50.598977+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-19T04:38:50.598977+00:00'
[2023-12-19T04:38:57.259+0000] {sql.py:274} INFO - Executing: 
            COPY steal(年,月,日,日期,分局,所,經度,緯度,種類)
            FROM 'D:/Airflow-docker/dags/data/steal_ETL.csv' DELIMITER ',' CSV HEADER;
            
[2023-12-19T04:38:57.265+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T04:38:57.270+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T04:38:57.305+0000] {sql.py:418} INFO - Running statement: 
            COPY steal(年,月,日,日期,分局,所,經度,緯度,種類)
            FROM 'D:/Airflow-docker/dags/data/steal_ETL.csv' DELIMITER ',' CSV HEADER;
            , parameters: None
[2023-12-19T04:38:57.308+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 280, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 392, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 423, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.DatetimeFieldOverflow: date/time field value out of range: "29019004"
HINT:  Perhaps you need a different "datestyle" setting.
CONTEXT:  COPY steal, line 683, column 日期: "29019004"

[2023-12-19T04:38:57.315+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=steal_etl_storeToPostgres, task_id=insert_sql, execution_date=20231219T043850, start_date=20231219T043856, end_date=20231219T043857
[2023-12-19T04:38:57.324+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 386 for task insert_sql (date/time field value out of range: "29019004"
HINT:  Perhaps you need a different "datestyle" setting.
CONTEXT:  COPY steal, line 683, column 日期: "29019004"
; 1602)
[2023-12-19T04:38:57.340+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-19T04:38:57.353+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
